src/library/autodj/autodjprocessor.cpp

```diff
@@ -625,7 +629,11 @@ void AutoDJProcessor::crossfaderChanged(double value) {
                     }
                     pToDeck->play();
                 } else {
+                    // TODO Is this also only possible if the user changed the
+                    // deck orientation?
```

Comment:
This happens if the track in the To Deck has been ejected or shorter than 200 ms.

Reply:
Okay, I understood the above condition, and it seems 'eject' and a manually loaded, very short track could be the only cause, since IIUC tracks shorter than `kMinimumTrackDurationSec` are skipped https://github.com/mixxxdj/mixxx/blob/0d5b4a79388afdb74cabc4eb19340844fb148b1c/src/library/autodj/autodjprocessor.cpp#L1223-L1228

If  AutoDJ stops unexpectedly a popup would be helpful to notify users.
But if that happens after user interaction (eject) we don't need it IMO